OpenRA.Mods.Common/Traits/Player/ResourceStorageWarning.cs

```diff
@@ -34,7 +34,7 @@ public class ResourceStorageWarning : ITick
 		readonly ResourceStorageWarningInfo info;
 		readonly PlayerResources resources;
 
-		int nextSiloAdviceTime = 0;
+		long lastSiloAdviceTime;
```

Comment:
Likewise.

Reply:
Actually probably not in this case either. We don't want to annoy players in the first tick of missions.